narayana/ArjunaJTA/jta/tests/classes/org/jboss/narayana/rts/EmptyTxnTest.java

```diff
@@ -50,15 +49,19 @@ public static void tearDown() throws IOException {
 
     @Benchmark
     public void testEmptyTxn(Blackhole bh) throws IOException {
-        bh.consume(beginTx());
+        Set<Link> links = beginTx();
         // make two service calls for comparison with RTSTests#testTxn
         bh.consume(sendRequest());
         bh.consume(TxnHelper.endTxn(txnClient, links));
     }
 
-    public boolean beginTx() throws IOException {
-        links = TxnHelper.beginTxn(txnClient, TXN_URL);
-        return true;
+    @Test
+    public void testEmptyTxn() throws IOException {
```

Comment:
Actually I added that test in order to have tha testEmptyTxn method tested also with junit, and not only with jmh. In fact it is not always easy for the developer to check that the jmh tests run fine because those tests are not executed in the maven lifecycle.

Reply:
I'll resolve this question since it is continued below.